### ISSUE TITLE
"Kab-omb" model from "New Super Mario Bros." not working

### ISSUE THREAD

**Issue by mariomadproductions** [scurest/apicula#1]
[hanabi.nsbmd](https://drive.google.com/file/d/0B_yBl-184usmMTcyLWpXcDRiWWs/view?usp=sharing)
[hanabi.nsbca](https://drive.google.com/file/d/0B_yBl-184usmcVIxa24wT2lNQXM/view?usp=sharing)

```
[...]apicula>apicula view hanabi.nsbmd
Found 1 model.
Found 0 animations.
ERROR:apicula: error: Error(
    Msg(
        "sanity check failed: num_terms <= 4"
    ),
    State {
        next_error: None,
        backtrace: None
    }
)
```

**Comment by scurest** [comment]
Thanks for the bug report!

I actually have a fix for this (b515a62), but hadn't pushed it. I've done so now; it should be in latest pre-built binary. Can you tell me if this fixes your model?

(Initialially I had thought it was still bugged because the model looks weird from the front. But NSMB is a side-scroller, and it looks okay from the right side.)

![both](https://cloud.githubusercontent.com/assets/11024420/24580322/bd18e4a2-16cb-11e7-80c4-4fe519ec6372.png)

**Comment by mariomadproductions** [comment]
It seems to work fine now. Thanks.
